src/Styles.res

```diff
@@ -279,8 +278,6 @@ let linkPillBox = style(.[
 ])
 let linkPillText = style(.[margin(px(10)), marginBottom(px(1)), color(hex("fff"))])
 
-let betaBanner = style(.[position(#absolute), left(#px(0)), top(#px(0)), width(#px(80))])
```

Comment:
Removed beta banner

Reply:
#px <- new syntax? rescript?

Reply:
Yeah, it is the new syntax, it used to be `px (for polymorphic variants).